walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,97 @@
 
 package com.google.sps;
 
+import java.io.*;
```

Comment:
Generally avoid wildcard imports as they can cause namespace cluttering and collisions (and they don't save that much typing :)

Not sure about other styleguides, but google's bans it:
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Same comment for `stream.*` in Event.java.